pkg/services/rds/update.go

```diff
@@ -42,6 +42,19 @@ func (b *RDSBroker) Update(instanceID string, details brokerapi.UpdateDetails, a
 	// Log InstanceDetails
 	b.Logger.Debug(fmt.Sprintf("rds instance in back database: %v", models.ToJson(ids)))
 
+	// sync and check status whether allowed to update
+	instance, err, serviceErr := SyncStatusWithService(b, instanceID, details.ServiceID, details.PlanID, ids.TargetID)
+
+	if err != nil || serviceErr != nil {
+		return brokerapi.UpdateServiceSpec{}, fmt.Errorf("sync status failed. error: %s, service error: %s", err, serviceErr)
+	}
+	if instance.Status != "ACTIVE" {
+		return brokerapi.UpdateServiceSpec{},
+			brokerapi.NewFailureResponse(
+				fmt.Errorf("Can only update rds instance in ACTIVE, but in: %s", instance.Status),
+				422, "Can only update rds instance in ACTIVE")
```

Comment:
you can use `http.StatusUnprocessableEntity` instead of `422`, looks more reasonable.

Reply:
Thank you